backends/gstreamer/lib.rs

```diff
@@ -88,6 +101,29 @@ impl Backend for GStreamerBackend {
     fn create_videoinput_stream(&self, set: MediaTrackConstraintSet) -> Option<MediaStreamId> {
         media_capture::create_videoinput_stream(set)
     }
+
+    fn can_play_type(&mut self, media_type: &str) -> SupportsMediaType {
```

Comment:
can_play_type() is a global backend function? or does it just belong to the player's API?

I am, actually, not sure.

Reply:
This will be useful for [MediaSource.isTypeSupported](https://developer.mozilla.org/en-US/docs/Web/API/MediaSource/isTypeSupported) as well, so it's ok to have it as a global backend function.